README.md

```diff
@@ -282,6 +282,39 @@ The list of image names may be changed with a command like:
 [info] List(buoyantio/linkerd:0.0.10-SNAPSHOT, gcr.io/gce-project/linkerd:v0.0.10-SNAPSHOT)
 ```
 
+#### DCOS ####
+
+Namerd supports a DCOS-specific configuration. When used in conjunction with
+namerd's `io.buoyant.namerd.storage.experimental.zk` dtab storage, this
+configuration provides bootstrapping of the ZooKeeper `pathPrefix`, along with
+a default dtab.
+
+##### Run locally #####
+
+This executes only the namerd-dcos-bootstrap command, it does not boot namerd.
+
+```bash
+$ ./sbt "namerd/dcos:run-main io.buoyant.namerd.DcosBootstrap namerd/examples/zk.yaml"
```

Comment:
i find this personally helpful for remembering how to run this stuff. it's possible we could move it into a separate file if we want to keep the README lean, but i'd like it documented somewhere.